ext/calendar/easter.c

```diff
@@ -115,18 +115,15 @@ static void _cal_easter(INTERNAL_FUNCTION_PARAMETERS, zend_long gm)
     ZVAL_LONG(return_value, result);
 }
 
-/* {{{ proto int easter_date([int year])
-   Return the timestamp of midnight on Easter of a given year (defaults to current year) */
+
```

Comment:
Does it make sense to drop functions' descriptions?

Reply:
Yes, I also wasn't exactly sure. But it seems that neither these descriptions are that useful. But

Reply:
No.

Reply:
IMHO, these "protos" should be removed, because they are just comments, and as such not rarely out-dated. Also, I believe these have been introduced before we had ZPP, and as such were useful in their own right. Furthermore, they are no longer used to generate the docs, to my knowledge. However, all that might better be discussed [on the mailing list](https://externals.io/message/110028).

Reply:
Right, I feel like proto comments are more wrong than correct nowadays :) We update the stubs, but we don't update the proto comments...

It might make sense to keep the description comment though.

Reply:
I'll have to redo the changes because many unintended new lines sneaked in despite my efforts, so I'll try to be considerate to those comments